internal/config/features.go

```diff
@@ -23,6 +23,8 @@ const (
 	FeatureMOFED    = featureName("mofed")
 	FeatureNVSWITCH = featureName("nvswitch")
 	FeatureGDRCopy  = featureName("gdrcopy")
+
+	FeatureDisableIMEXChannelCreation = featureName("disable-imex-channel-creation")
```

Comment:
Why the newline?

Reply:
I sometimes introduce newlines to prevent additional whitespace changes due to aligning the `=` in the assignment. Can roll it back if required.

Reply:
OK, I actually like it when its changed to line up the `=`. It makes it more readable in the code (even if the diff isn't as obvious). But I don't feel strongly.

Reply:
I have updated this now. Thanks for the feedback.